dynamodb/src/test/scala/zio/dynamodb/codec/CodecRoundTripSpec.scala

```diff
@@ -278,44 +278,36 @@ object CodecRoundTripSpec extends DefaultRunnableSpec with CodecTestFixtures {
           }
       }
     },
-    testM("any Map") {
-      checkM(SchemaGen.anyMapAndValue) {
+    test("any Map") {
+      check(SchemaGen.anyMapAndValue) {
         case (schema, value) =>
           assertEncodesThenDecodes(schema, value)
       }
-    },
-    testM("any Set") {
```

Comment:
Is this an intentional removal?

Reply:
It was at the time, some zio-aws or schema related reason. Brought them back now.